src/simulations/simulator.py

```diff
@@ -162,9 +162,9 @@ def run_one(self):
       sketch.add_ids(campaign_ids)
       sketches.append(sketch)
 
-    # Optionally noise the sketches.
-    sketch_noiser = self.estimator_config.sketch_noiser
-    if sketch_noiser:
+    # Optionally noise the sketches if it exists in the estimator_config.
+    if hasattr(self.estimator_config, 'sketch_noiser'):
```

Comment:
Can you add a test for this behavior?

Reply:
The test already exists. These changes fix existing tests.